### PR TITLE
Update VigLink.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/VigLink.com.xml [hunk 1]
```diff
@@ -58,29 +60,19 @@
...
 		but not forward slash:
 					-->
         <rule from="^http://blog\.viglink\.com/+"
-                to="https://www.viglink.com/blog" />
+                to="https://www.viglink.com/blog/" />
 
 		<test url="http://blog.viglink.com//" />
 
         <rule from="^http://support\.viglink\.com/"
                 to="https://viglink.zendesk.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/VigLink.com.xml]: WFM
Please also remove the exclusions and tests.

File: src/chrome/content/rules/VigLink.com.xml [hunk 2]
```diff
@@ -58,29 +60,19 @@
...
-	<!--securecookie host="^publishers\.viglink\.com$" name="^(?:AWSELB|WEBROUTEID)$" /-->
-
-	<securecookie host="." name="." />
-
+	<securecookie host=".+" name=".+" />
 
 	<!--	Redirect keeps path, args,
 		but not forward slash:
 					-->
         <rule from="^http://blog\.viglink\.com/+"
```
Comment (J0WI) [inline src/chrome/content/rules/VigLink.com.xml]: The `+` is not needed.

File: src/chrome/content/rules/VigLink.com.xml [hunk 3]
```diff
@@ -58,29 +60,19 @@
...
-                to="https://www.viglink.com/blog" />
+                to="https://www.viglink.com/blog/" />
 
 		<test url="http://blog.viglink.com//" />
```
Comment (J0WI) [inline src/chrome/content/rules/VigLink.com.xml]: Can be removed.

File: src/chrome/content/rules/VigLink.com.xml [hunk 4]
```diff
@@ -11,7 +12,7 @@
 
 		- blog *
```
Comment (J0WI) [inline src/chrome/content/rules/VigLink.com.xml]: `info.viglink.com`

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/VigLink.com.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<!--
+	Self-signed:
+		- blog
```
Comment (J0WI) [inline src/chrome/content/rules/VigLink.com.xml]: This needs a rule to be rewritte to `https://www.viglink.com/blog`

Comment (J0WI) [inline src/chrome/content/rules/VigLink.com.xml]: See also `info.viglink.com`

**Comment by koops76** [comment]
@J0WI Done.

**Comment by koops76** [comment]
@J0WI `^` is not accessible either over HTTP or HTTPS. `www` is an empty Heroku app. Should we secure it anyway? I added a rule which redirects from `^` to `www` since many people do not type `www` in the URL (like me).

**Pull request merged**